client/tests/Spotlight.test.js

```diff
@@ -13,4 +14,28 @@ describe('Unit Test for <Spotlight /> Component', () => {
     const component = (<Spotlight numPhotos={13} />);
     expect(component.props.numPhotos).toEqual(13);
   });
+  
+  it('should invoke shiftSlide on user click of spotlight image', () => {
+    const testPhoto = {photoId: 2, url: 'testUrl', caption: 'testCaption'};
+    const shiftSlide = sinon.spy();
```

Comment:
If you would like to limit your dependencies you can also monitor whether a function has been called using a mock function in Jest: https://facebook.github.io/jest/docs/en/mock-functions.html